packages/uikit/src/components/Link/ScanLink.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useMemo } from "react";
+import Link from "./Link";
+import OpenNewIcon from "../Svg/Icons/OpenNew";
+import BscScanIcon from "../Svg/Icons/BscScan";
+import AptosIcon from "../Svg/Icons/Aptos";
+import { LinkProps } from "./types";
+
+interface ScanLinkProps extends Omit<LinkProps, "external" | "showExternalIcon"> {
+  forceChain?: "BSC" | "APTOS" | string;
```

Comment:
Can we design the interface in a more flexible way I mean chain-wise? Cuz for `Aptos` it's quite a special case. I'm thinking of the props like this

```typescript
interface ScanLinkProps extends Omit<LinkProps, "external" | "showExternalIcon"> {
  icon: ReactNode;
  chainId?: ChainId
}
```

So for evm compatible chains we could have a icon map for future maintenance, while for aptos we could pass in the icon node explicitly
```typescript
const icons = {
  [ChainId.BSC]: BscScanIcon,
}
```

What do you think?